yaksh/models.py

```diff
@@ -1129,6 +1154,113 @@ def has_lessons(self):
                 break
         return status
 
+    @classmethod
+    def _create_object_from_dict(cls, model_obj, model_type):
+        model_class = get_model_class(model_type)
+        new_obj, obj_create_status = model_class.objects.get_or_create(**model_obj)
+        return new_obj, obj_create_status
+
+    @classmethod
+    def load_yaml(cls, filepath, user):
+        file_list = {}
```

Comment:
```file_list``` is initialized as dict
So variable name can simply be ```files```

Reply:
Done